### PR TITLE
Polish basics tutorial

### PR DESCRIPTION
**Pull request by hanna-skryl**
- [x] fixed typos
- [x] adopted class names
- [x] added comments to the code snippets

### CODE REVIEW

File: apps/demos/src/app/features/tutorials/basics/2-input-bindings/input-bindings.start.component.ts [hunk 1]
```diff
@@ -101,7 +102,7 @@ export class InputBindingsStart extends RxState<ComponentState> implements OnIni
   @Input()
   set refreshInterval(refreshInterval: number) {
     if (refreshInterval > 4000) {
-      this._refreshInterval = refreshInterval;
+      this._refreshInterval = refreshInterval; // Partially update the state with the state slice
```
Comment (BioPhoton) [inline apps/demos/src/app/features/tutorials/basics/2-input-bindings/input-bindings.start.component.ts:105]: If there is a "helper comment" for the person doing the tutorial mostly it's a good idea to put the comment above.
Also a nice thing we can do here is use emoticons to highlight a specific piece of code e.g. 👇
```suggestion
      this._refreshInterval = refreshInterval; // Partially update the state with the state slice
```

**Review by BioPhoton — CHANGES_REQUESTED** [review]
Hi, @hanna-skryl thanks soo much for this nice PR! ♥
If you could do a small change and move the comments above and also use emojis to highlight the sections and make the whole text visually more appealing.

**Comment by hanna-skryl** [comment]
@BioPhoton The pull request has been updated. Please review it at your convenience.

**Review by BioPhoton — APPROVED** [review]

**Comment by BioPhoton** [comment]
Thanks a lot for this contribution!! <3

**Pull request merged**